src/Machete.HL7Schema/Version26.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Machete.HL7Schema.V26
+{
+    public interface Version26
```

Comment:
makes sense. I even thought this as well while doing this but the model generator is not accessible. Can you please add it and delete that file.